results.csv

```diff
@@ -1,18 +1,33 @@
-VehicleID,Delayed in hours
```

Comment:
Ok, ein Fehler ist die Sache wohl aber nicht. Git verfolgt alle Dateien die einmal rein commited wurden. Das Verfolgen von den Änderungen geht auch dann weiter, wenn die Datei mitlerweile eigentlich in der .gitignore steht. Wenn man einmal die Datei commited und diese dann aber in der .gitignore steht hat man quasi einen Widerspruch. Git nimmt dann die sichere Variante und verfolgt die Datei immer weiter. 

Das weitere Verfolgen kann man durch löschen des Cache beenden.

![grafik](https://user-images.githubusercontent.com/28202401/29242746-9d3f636a-7f92-11e7-8bf8-6dfff5a2581a.png)
(Ausschnitt von [hier](https://stackoverflow.com/questions/1139762/ignore-files-that-have-already-been-committed-to-a-git-repository))